nsiqcppstyle_reporter.py

```diff
@@ -62,27 +63,28 @@ def ReportSummaryToScreen(analyzedFiles, nsiqcppstyle_state, filter, ciMode) :
     buildQuality = 0
     if fileCount != 0 :
         buildQuality = (fileCount - violatedFileCount) * 100.0 / fileCount 
-    print "\n"
-    print ("=================================== Summary Report ===================================");
-    print (" ** Total Available Rules     : %d" % nsiqcppstyle_rulemanager.ruleManager.availRuleCount)
-    print (" ** Total Applied Rules       : %d" % len(nsiqcppstyle_state.checkers))
-    print (" ** Total Violated Rules      : %d" % len(nsiqcppstyle_state.errorPerChecker.keys()))
-    print (" ** Total Errors Occurs       : %d" % nsiqcppstyle_state.error_count)
-    print (" ** Total Analyzed Files      : %d" % len(analyzedFiles))
-    print (" ** Total Violated Files Count: %d" % violatedFileCount)
-    print (" ** Build Quality             : %.2f%%" % buildQuality)
-    if not ciMode :
-        print ("\n================================ Violated Rule Details ===============================")
+    console.Out.Ci("\n")
+    console.Out.Ci("=================================== Summary Report ===================================")
+    console.Out.Ci(" ** Total Available Rules     : %d" % nsiqcppstyle_rulemanager.ruleManager.availRuleCount)
+    console.Out.Ci(" ** Total Applied Rules       : %d" % len(nsiqcppstyle_state.checkers))
+    console.Out.Ci(" ** Total Violated Rules      : %d" % len(nsiqcppstyle_state.errorPerChecker.keys()))
+    console.Out.Ci(" ** Total Errors Occurs       : %d" % nsiqcppstyle_state.error_count)
+    console.Out.Ci(" ** Total Analyzed Files      : %d" % len(analyzedFiles))
+    console.Out.Ci(" ** Total Violated Files Count: %d" % violatedFileCount)
+    console.Out.Ci(" ** Build Quality             : %.2f%%" % buildQuality)
+    if console.IsLevelDisplayed(console.Level.Info) :
```

Comment:
This condition check isn't required anymore 😄

Reply:
I've kept it to avoid unnecessary computation within the for loops. Are you sure you want to remove this check ?

Reply:
Oh shoot my bad. Didn't see the following lines